app/models/mechanic.rb

```diff
@@ -0,0 +1,8 @@
+class Mechanic < ApplicationRecord
+  has_many :ride_mechanics
+  has_many :rides, through: :ride_mechanics
+
+  def working_ride_list
```

Comment:
I don't think that you will need this method. I am assuming that you are trying to get the names of all the rides that a mechanic is working on for US 1. You can iterate over `@mechanic.rides` in your view.